src/Server.cpp

```diff
@@ -129,71 +128,17 @@ void Server::handleConnection(Connection conn, size_t &i)
 			return;
 		}
 	}
-	// It should be double "\r\n" to separate the headers from the body
-	std::string httpRequestString = conn.getHeaders() + conn.getBody();
-	std::cout << "Received HTTP request: " << std::endl << httpRequestString << std::endl;
-	HTTPRequest request(httpRequestString.c_str());
-	std::cout << request.getStatusCode() << std::endl;
-	std::cout << "Received HTTP request: " << std::endl << httpRequestString << std::endl;
 
-	// test to execute the python script (see: https://www.tutorialspoint.com/python/python_cgi_programming.htm)
-	const char *argv[] = {"./cgi-bin/hello_py.cgi", NULL};
-	// const char* argv[] = { "./cgi-bin/thirty_py.cgi", NULL };
-	// const char* argv[] = { "./cgi-bin/hello.cgi", NULL };
+	// NOTE: end of buffering/parsing part, start of router
 
-	// std::string response;
 	Router router;
+	std::string httpRequestString = conn.getHeaders() + conn.getBody();
+	HTTPRequest request(httpRequestString.c_str());
+	std::string responseString;
 	HTTPResponse response;
```

Comment:
a response object is declared here as well.

Reply:
answered [here](https://github.com/dantol29/webserver/pull/147#discussion_r1569447552)